responsive-web-design/intermediate-css-concepts.md

```diff
@@ -0,0 +1,65 @@
+<div align="center">
+    <img src="../images/css.jpg" alt="Logo" height="170" align="center">
+    <h1 align="center">Intermediate CSS Concepts</h1>
+</div>
+
+## 📝 Description
+> CSS is a stylesheet language that allows you to define the presentation of elements in a markup language like HTML. CSS separates structure from presentation, allowing you to style your HTML elements.
+
+## 📋 Prerequisites
+- [ ] [Introduction to HTML and HTML5](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-html-and-html5.md).
+- [ ] [Introduction to CSS and Basic CSS](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-css-and-basic-css.md).
+- [ ] [Introduction to CSS Colors](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-css-colors.md).
+- [ ] [Introduction to HTML Forms](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-html-forms.md).
+- [ ] [Introduction to CSS Box Model](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-css-box-model.md).
+- [ ] [Introduction to CSS Flexbox](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-css-flexbox.md).
+- [ ] [Introduction to Typography](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-typography.md).
+- [ ] [Introduction to Accessibility](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/introduction-to-accessibility.md).
+- [ ] [Introduction to CSS Selectors](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/intrduction-to-css-selectors.md).
+
+## 🎯 Learning Objectives
+- [ ] Understand the basic structure of CSS.
+- [ ] Basic understanding of CSS selectors.
+- [ ] Basic understanding of CSS properties.
+- [ ] Basic understanding of CSS properties values.
+
+## 📹 Video
+
+- Please watch the following videos:
+    - [CSS](https://www.youtube.com/watch?v=1Rs2ND1ryYc)
+
+    [![Video](https://img.youtube.com/vi/1Rs2ND1ryYc/0.jpg)](https://www.youtube.com/watch?v=1Rs2ND1ryYc)
+
+## 🔧 Instructions
+- If you don't have an account on [freecodecamp](freecodecamp.org), please create one and familiarize yourself with the platform.
```

Comment:
Kindly check the `freecodecamp` link it navigates me to [github.com/{Kick-Start}/responsive-web-design/freecodecamp.org](https://github.com/Kick-StartDev/web-development-basic-curriculum/blob/responsive-web-design/responsive-web-design/freecodecamp.org) and has a 404 error

![404 error](https://github.com/Kick-StartDev/web-development-basic-curriculum/assets/87229464/e4340b1b-1c31-47cf-9604-b494a8a1de82)
I think you have this error because you forgot to use the `https://` before the URL
``` suggestion
- If you don't have an account on [freecodecamp](https://www.freecodecamp.org), please create one and familiarize yourself with the platform.
```
**_Note: All files have the same problem_**